references and rewording around BIOS (#105)
linuxboot/book: src/history.md

## BIOS

[BIOS](https://en.wikipedia.org/wiki/BIOS) is the good old, inscrutable way of
initializing a hardware platform in the pre-UEFI days. It's a binary blob with
no standardized structure, that is responsible for initializing CPU and memory,
and jumping to a hard-coded position on the MBR of the first disk drive.

Starting around 2000, BIOS has been largely replaced by the standardized
[UEFI](https://en.wikipedia.org/wiki/UEFI).
Many UEFI implementations still offer a BIOS compatibility mode called CSM
(Compatibility Support Module), which makes it behave like an old BIOS.

BIOS is also a misused term for system firmware nowadays. You may still hear

> "BIOS" in reference to system firmware, such as UEFI or even LinuxBoot. However,

Reviewer: ```suggestion
"BIOS" may be used to refer to system firmware, such as UEFI or even LinuxBoot. However,
```

Author: Yea, I've reworded a bit further to get rid of redudancy and add more clarity instead. :)
